javascript/builtins/intl/DurationFormat.json

```
@@ -0,0 +1,229 @@
+{
+  "javascript": {
+    "builtins": {
+      "Intl": {
+        "DurationFormat": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Intl/DurationFormat",
+            "spec_url": "https://tc39.es/proposal-intl-duration-format",
```

Build fails because we always want a deeplink to the specific feature.

```suggestion
            "spec_url": "https://tc39.es/proposal-intl-duration-format/#durationformat-objects",
```